.github/workflows/tests.yml

```diff
@@ -0,0 +1,34 @@
+on:
+  pull_request:
+  push:
+    branches:
+      - master
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Clone govuk-content-schemas
+        uses: actions/checkout@v2
+        with:
+          repository: alphagov/govuk-content-schemas
+          ref: deployed-to-production
+          path: tmp/govuk-content-schemas
+      - uses: ruby/setup-ruby@v1
+      - uses: actions/cache@v1
+        with:
+          path: tmp/bundle
+          key: ${{ runner.os }}-gems-${{ hashFiles('**/Gemfile.lock') }}
+          restore-keys: |
```

Comment:
Could you help me understand what this is for? It seems to be a fallback, but I'm not sure where it came from.

Reply:
The short answer is, I don't know. I copied all the caching stuff from the [actions/cache Bundler example](https://github.com/actions/cache/blob/master/examples.md#ruby---bundler), which has this included. The docs say that it's "an ordered list of keys to use for restoring the cache if no cache hit occurred for key", but that's not terribly enlightening as to _why_.